docs/changelogs/v2.1.1.md

```diff
@@ -1,3 +1,22 @@
 _Release Date: 8/04/2025_
 
 # v2.1.1
+
+- [Map changes](#map-changes)
+
+## Map changes
+
+Ravine
+
+- Moved beacon **7** blocks back from the cliff.
+- Moved Spawn **2** blocks back from the beacon.
```

Comment:
please lowercase the S in Spawn

Reply:
Please specify the exact number the spawn point moved so there is less confusion

Reply:
change "back" to "away"